frontend/src/pages/projects/components/StorageSizeBars.tsx

```diff
@@ -43,8 +43,12 @@ const StorageSizeBar: React.FC<StorageSizeBarProps> = ({ pvc }) => {
   let inUseRender: React.ReactNode;
   if (error) {
     inUseRender = (
-      <Tooltip removeFindDomNode content={`Unable to get storage data. ${error.message}`}>
-        <ExclamationCircleIcon color="var(--pf-global--danger-color--100)" />
+      <Tooltip removeFindDomNode content={`Unable to get storage data.`}>
```

Comment:
Was removing `error.message` from the tooltip intentional? Just checking :)

Reply:
Nice catch, reverted